src/oca_github_bot/manifest.py

```diff
@@ -85,6 +87,18 @@ def set_manifest_version(addon_dir, version):
         f.write(manifest)
 
 
+def is_maintainer(username, addon_dirs):
+    for addon_dir in addon_dirs:
+        try:
+            manifest = get_manifest(addon_dir)
+        except NoManifestFound:
+            return False
+        maintainers = manifest.get("maintainers", [])
+        if username not in maintainers:
+            return False
+    return True
```

Comment:
@sbidoul Does this mean that if you are the maintainer of a module in a repo, you can merge any module of that repo?

Reply:
no, only pr that only touch addons where you are maintainer in the target branch

Reply:
see `_user_can_merge()`